dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+- `152 - ATTIC ROOM - BATHROOMS`
+- `152.1 - ATTIC ROOM - BATHROOMS - Attic area being converted from unfinished to finished (including new bathrooms in the basement area)`
+- `152.2 - ATTIC ROOM - BATHROOMS - Attic finished area being remodeled`
+- `153 - FIREPLACE-CENTRAL AIR CONDITIONING`
+- `153.1 - FIREPLACE-CENTRAL AIR CONDITIONING - fireplace`
+- `153.2 - FIREPLACE-CENTRAL AIR CONDITIONING - central air conditioning`
+- `153.3 - FIREPLACE-CENTRAL AIR CONDITIONING - replacement central air conditioning`
+- `154 - OTHER - REMODELING`
+- `154.1 - OTHER - REMODELING - renovation`
+- `154.11 - OTHER - REMODELING - water taps - (should be verified that they are not for new construction)`
+- `154.2 - OTHER - REMODELING - general repairs`
+- `154.3 - OTHER - REMODELING - elevators`
+- `154.4 - OTHER - REMODELING - handicap ramps`
+- `154.5 - OTHER - REMODELING - fire & flood alarms`
+- `154.6 - OTHER - REMODELING - canopies`
+- `154.7 - OTHER - REMODELING - water taps - (should be verified that they are not for new construction)`
+- `154.8 - OTHER - REMODELING - fire & flood alarms`
+- `154.9 - OTHER - REMODELING - canopies`
+- `171 - MAJOR IMPROVEMENT - WRECK`
+- `171.1 - MAJOR IMPROVEMENT - WRECK - major demolition`
+- `171.2 - MAJOR IMPROVEMENT - WRECK - house moving`
+- `172 - MINOR IMPROVEMENT - WRECK`
+- `172.1 - MINOR IMPROVEMENT - WRECK - garages`
+- `172.2 - MINOR IMPROVEMENT - WRECK - sheds`
+- `172.3 - MINOR IMPROVEMENT - WRECK - debris removal`
+- `191 - MAJOR IMPROVEMENT - BURNOUT`
+- `192 - MINOR IMPROVEMENT - BURNOUT`
+- `192.1 - MINOR IMPROVEMENT - BURNOUT - garage`
+- `192.2 - MINOR IMPROVEMENT - BURNOUT - shed`
+- `211 - NEW BUILDING`
+- `212 - ADDITIONS`
+- `213 - OTHER - MAJOR NEW CONSTRUCTION`
+- `213.1 - OTHER - MAJOR NEW CONSTRUCTION - foundations`
+- `213.2 - OTHER - MAJOR NEW CONSTRUCTION - build-outs`
+- `213.3 - OTHER - MAJOR NEW CONSTRUCTION - cement pads`
+- `213.4 - OTHER - MAJOR NEW CONSTRUCTION - loading dock`
+- `231 - SWIMMING POOL - TENNIS COURT`
+- `231.1 - SWIMMING POOL - TENNIS COURT - tennis courts`
+- `231.2 - SWIMMING POOL - TENNIS COURT - swimming pools`
+- `231.3 - SWIMMING POOL - TENNIS COURT - recreation improvements`
+- `232 - DRIVEWAYS - PATIOS`
+- `232.1 - DRIVEWAYS - PATIOS - rock`
+- `232.2 - DRIVEWAYS - PATIOS - asphalt`
+- `232.3 - DRIVEWAYS - PATIOS - cement`
+- `232.4 - DRIVEWAYS - PATIOS - crushed rock`
+- `232.5 - DRIVEWAYS - PATIOS - pavers`
+- `233 - COMMERICAL FENCING`
+- `233.1 - FENCING - wrought iron`
+- `233.2 - FENCING - security`
+- `233.3 - FENCING - wood`
+- `233.4 - FENCING - chain link`
+- `234 - GARAGE-BARN-BUTLER-TRASH ENCLOSURES BUILDING`
+- `234.1 - GARAGE-BARN-BUTLER BUILDING - garages`
+- `234.2 - GARAGE-BARN-BUTLER BUILDING - barns`
+- `234.3 - GARAGE-BARN-BUTLER BUILDING - butler buildings`
+- `234.4 - Trash Enclosures`
+- `235 - OTHER-MINOR NEW CONSTRUCTION`
+- `235.1 - OTHER-MINOR NEW CONSTRUCTION - construction trailers`
```

Comment:
**nitpick (non-blocking):** Again for consistency:

```suggestion
- `235.1 - OTHER - MINOR NEW CONSTRUCTION - construction trailers`
```

And the same for all below.

Reply:
Fixed in 7d94c97522a10d4303a21f3e5634ae39c567aa4b.